lesson06/assignment/accordion.html

```diff
@@ -0,0 +1,124 @@
+                                <div class="modal-dialog modal-lg">
+                                    <div class="modal-content">
+                                        Weathering with You is a Japanese animated film written and directed by <br>Makoto Shinkai and produced by CoMix Wave Films, following the 2016 <br>film Your Name. It was released in Japan on July 19, 2019, starring <br>Kotaro Daigo and Nana Mori. Wikipedia<br>
+                                        Initial release: July 19, 2019 (Japan)<br>
+                                        Director: Makoto Shinkai<br>
+                                        Japanese: 天気の子<br>
+                                        Story by: Makoto Shinkai<br>
+                                        Screenplay: Makoto Shinkai<br>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-header">
+                        <a href="#three"
+                            data-parent="#accordion" data-toggle="collapse">
+                            <img src="img/anime-3.jpg" alt="">
+                        </a>
+                    </div>
+                    <div id="three" class="collapse">
+                        <div class="card-body">
+                            <h3 class="display-4" data-toggle="modal" data-target=".bd-example-modal-lg">My Neighbor Totoro</h3>
+                            <div class="modal fade bd-example-modal-lg" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel" aria-hidden="true">
+                                <div class="modal-dialog modal-lg">
+                                    <div class="modal-content">
+                                        This acclaimed animated tale by director Hayao Miyazaki follows schoolgirl<br>Satsuke and her younger sister, Mei, as they settle into an old country house<br> with their father and wait for their mother to recover from an illness in an area hospital.<br>
+                                        Release date: May 7, 1993 (USA)<br>
+                                        Director: Hayao Miyazaki<br>
+                                        Featured song: My Neighbor Totoro<br>
+                                        Japanese: となりのトトロ<br>
+                                        Cast: Shigesato Itoi, Chika Sakamoto, Sumi Shimamoto..<br>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <h2 class="display2 bryce">Bryce please help me to get rid of secondary 2 boxes showing up when clicking on the main one -.-"</h2>
```

Comment:
Right now you've got the `.card`s resting in your flex container. When you trigger an accordion, the cards will expand to fill the content -- and since there is a border on the overall `.card`, we see the border appear on all of them. You'll want to adjust the `align-items` of your `#accordion` to `baseline` to get what you're looking for.